django_project/ford3/templates/campus.html

```diff
@@ -32,6 +32,7 @@ <h4 class="mt-3 pb-2">1. Details</h4>
         </div>
         <div class='row py-4'>
             <div class='col-md-12'>
+	            <p><b>Name</b> {{ campus.name|is_none }}</p>
```

Comment:
![image](https://user-images.githubusercontent.com/34506346/59715840-03f70680-9215-11e9-9470-d913ea4e0545.png)
for peace of mind xD